createx/src/components/Button/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Button = ({ children, to, onClick, className = '' }) => {
+  const baseClasses = 'px-6 py-2 bg-orange-500 text-white rounded-[0.6125rem] text-base font-bold hover:bg-orange-600 transition-all duration-300 inline-block font-kagitingan text-xl text-center';
+  
+  if (to) {
+    return (
+      <Link to={to} className={`${baseClasses} ${className}`}>
+        {children}
+      </Link>
+    );
+  }
+
+  return (
+    <button
+      onClick={onClick}
+      className={`${baseClasses} ${className}`}
+    >
+      {children}
+    </button>

```

Comment:
It is not a correct way to handle variants suing a if condition. We need to have a mapped state to map different style variants in Javascript state array so as to improve the process and also organize the variants properly using React Props.
```suggestion

export const ButtonVariant = {
  Primary: 'primary',
  Secondary: 'secondary',
  OutlinePrimary: 'outline-primary',
  OutlineSecondary: 'outline-secondary',
};
```

Use this mapping to map variants so that we could adapt easily if any variant comes in to picture for now only one variant is present so manage it and store all states: hover, focus and active ina organized manner and apply using single variant prop.

Also never use this of statement to render the Link Tag ( IT IS VERY BAD PRACTICE OF CDOE)

You should never merge 2 features in a atomic level component. @oyejateen